install/Docker/dockerfiles/Dockerfile.compositionService

```diff
@@ -14,7 +14,11 @@
 
 FROM odim:3.0 as build-stage
 
+
 FROM cs:1.0 as build-stage-cs
+COPY install/Docker/dockerfiles/build/compositionservice.sh .
```

Comment:
why these files are changed ?

Reply:
this changes is made for building the composition services

Reply:
commited the changes